specification/src/chapter2.adoc

```diff
@@ -2,9 +2,600 @@
+* Accelerators
+
+==== Risc-V ISA and non-ISA extensions
+
+Risc-V defines a number of security related extensions. These will be discussed in more detail later in this document. 
 
 === Adversarial model
 
-=== Security objectives
+For the purpose of this specification, the main goal of an adversary is to gain unauthorized access to _resources_ - memory, memory mapped devices, and execution state. For example, to access sensitive assets, to gain privileges, or to affect the control flow of a victim.
+
+In general, adversaries capable of mounting the following broad classes of attacks should be considered by system designers:
+
+* Logical +
+The attacker and the victim are both processes on the same system.
+
+* Physical +
+The victim is a process on a system, and the attacker has physical access to the same system. For example: probing, interposers, glitching, and disassembly.
+
+* Remote +
+The victim is a process on a system, and the attacker does not have physical or logical access to the system. For example, radiation or power fluctuations, or protocol level attacks on connected services.
+
+Attacks can be direct or indirect:
+
+* Direct +
+An adversary gains direct access to a resource belonging to the victim. For example: direct access to a memory location or execution state, or direct control of the control flow of a victim.
+
+* Indirect +
+An adversary can access or modify the content of a resource by a side channel. For example: by analyzing timing patterns of an operation by a victim to reveal information about data used in that operation, or launching row-hammer style memory attacks to affect the contents of memory owned by the victim.
+
+* Chained +
+An adversary is able to chain together multiple direct and indirect attacks to achieve a goal. For example, use a software interface exploit to affect a call stack, and use that to take redirect the control flow of a victim.
+
+This specification is primarily concerned with ISA level mitigations against logical attacks.
+
+Physical or remote attacks in general need to be addressed at system, protocol or governance level, and may require additional non-ISA mitigations. However, some ISA level mitigations can also help provide some mitigation against physical or remote attacks and this is indicated in the tables below.
+
+The required level of protection can vary depending on use case. For example, a HW RoT may have stronger requirements on physical resistance than other parts of an SoC.
+
+Finally, this specification does not attempt to rate attacks by severity, or by adversary skill level. Ratings tend to depend on use case specific threat models and requirements. 
+
+==== Logical
+
+[width=100%]
+[%header, cols="5,5,5,10,15,10"]
+|===
+| ID#
+| Attack   
+| Type  
+| Description
+| Risc-V mitigations 
+| Planned mitigations
+
+| CAT_NNN
+| Unrestricted access 
+| Direct +
+Logical
+| Direct access to unauthroized resources in normal operation.
+a| * Risc-V privilege levels
```

Comment:
```suggestion
a| * RISC-V (V)U/(V)S/S(H)/M privilege levels
```

Reply:
I would prefer to just keep "Risc-V privilege levels". The full definition of those is later, I don't think we need to keep enumerating that everywhere.